cmd/node/config/api.toml

```diff
@@ -49,10 +49,7 @@
         { Name = "/:address/esdt/:tokenIdentifier", Open = true },
 
         # /address/:address/nft/:tokenIdentifier/nonce/:nonce will return data of an nft esdt token for a given account, tokenID and nonce
```

Comment:
`data of a nft` instead `data of an nft`

Reply:
done